tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/attachment_options.py

```diff
@@ -9,22 +9,23 @@
 
 class AttachmentOptions(Base):
 
-    _attachment_options_locator = (By.CSS_SELECTOR, '#attachment-options[data-type="action"]')
-    _view_button_locator = (By.ID, 'attachment-options-view')
-    _header_locator = (By.CSS_SELECTOR, '.current gaia-header')
-    _cancel_button_locator = (By.ID, 'attachment-options-cancel')
+    _attachment_options_locator = (By.CSS_SELECTOR, 'form.visible[role="dialog"][data-subtype="menu"]')
+    _view_button_locator = (By.CSS_SELECTOR, 'button[data-l10n-id="view-attachment-image"]')
+    _cancel_button_locator = (By.CSS_SELECTOR, 'button[data-l10n-id="cancel"]')
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
-        element = self.marionette.find_element(*self._attachment_options_locator)
-        Wait(self.marionette).until(lambda m: element.location['y'] == 0)
+        Wait(self.marionette).until(lambda m: self.root_element.location['y'] == 0)
```

Comment:
Oops, my bad! If we put `self.root_element` in a Wait, we'll run a search every millisecond. We need to do it this way:
``` python
root_element = self.root_element
Wait(self.marionette).until(lambda m: root_element.location['y'] == 0)
``` 

Sorry for the confusion! :s